hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationRunnerImpl.java

```diff
@@ -341,7 +341,7 @@ private void handleOperationError(Operation operation, Throwable e) {
 
         operation.logError(e);
 
-        if (e instanceof ResponseNotSentException || !operation.returnsResponse()) {
+        if (e instanceof ResponseNotSentException || !operation.isRetryable()) {
```

Comment:
Thx Ali. Yes, I agree. 
I'm trying to find a minimal subset of changes to fix the PartitionCheckIfLoadedOperation. This change that I propose here will be reverted.